wrapper.js

```diff
@@ -74,22 +74,26 @@ var main = function(args) {
 	}
 
 	var doLint = function(filepath) {
+		//override for lint
```

Comment:
Just used this code for check js files in emacs. Thanks for eslint hint, maybe it is more interesting.
yes, transformWarning copied just for compatibility with jshint, I don't know what is it ) and don't want to break something